es/px-common.json

```diff
@@ -153,6 +153,16 @@
                 },
                 "recaptchaCheck": {
                     "title": "Un momento... solo se aceptan humanos."
+                },
```

Comment:
Are those expected as well?

Reply:
UX added changes to common, these are the new translations for the moderated card and needed for the R2QA. So it will be useful for that. Can't do two separate PRs due to how pushing from poeditor to gh works

Reply:
No problem, and since additions cannot mess something up LGTM!